source/_integrations/forecast_solar.markdown

```diff
@@ -135,4 +135,4 @@ instance:
 1. Browse to your Home Assistant instance.
 2. Go to **{% my integrations title="Settings > Devices & services" %}**.
 3. If multiple instances of {{ name }} are configured, choose the instance you want to configure.
-4. Select **Configure**.
+4. Select **Add**.
```

Comment:
I think this is also related to an already existing config entry, those related to the options flow?